app/controllers/api/providers_controller.rb

```diff
@@ -114,6 +114,15 @@ def change_password_resource(type, id, data = {})
       end
     end
 
+    def verify_credentials_resource(_type, _id, data = {})
```

Comment:
Is this part of your change?  It looks like it was added by https://github.com/ManageIQ/manageiq-api/pull/683/commits/5b61ddf5457231ceb827ac3ed9d7a89349b98c34

You should drop the merge commit and rebase this on master